remoulade/state/middleware.py

```diff
@@ -44,7 +44,7 @@ def _get_current_time(self):
 
     def after_enqueue(self, broker, message, delay):
         priority = broker.get_actor(message.actor_name).priority
-        group_id = message.options.get("group_info", {}).get("group_id")
+        group_id = self.get_option_value("group_info", broker, message, default={}).get("group_id")
```

Comment:
group info is linked to a particular group, it does not make sense at any other level than message (it's the case above too);